docker/addons/prometheus/grafana/example-dashboard.json

```diff
@@ -0,0 +1,451 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
```

Comment:
For the Up time metric change to when it is 0 to display red. You can change the value mapping.
Something like this 
![image](https://user-images.githubusercontent.com/28790446/231414035-fbf6dc14-5a0a-448b-9d97-8801d1c11659.png)